### PR TITLE
Fix custom keyboard height

### PR DESCRIPTION
**Pull request by mario**

### CODE REVIEW

**Comment by mario** [comment]
I still don't see a use case for this, but at least it now works as expected.

**Comment by mario** [comment]
Requires previous PR to be merged.

**Comment by mario** [comment]
Rebased.

**Comment by vanniktech** [comment]
@mario needs a rebase :/

**Comment by mario** [comment]
Ask @vanniktech :P

**Comment by mario** [comment]
Ready for review.

File: emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java [hunk 1]
```diff
@@ -397,9 +401,11 @@ private Builder(final View rootView) {
      * If height is not 0 then this value will be used later on. If it is 0 then the keyboard height will
```
Comment (rubengees) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: These docs are not correct anymore,should be something like "if height is greater than 0".

Comment (mario) [reply emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: Fixed.

**Comment by vanniktech** [comment]
Any opinion @rubengees ?

**Comment by mario** [comment]
Poking @rubengees once more :)

**Comment by mario** [comment]
Thanks folks!

**Pull request merged**